TweetBook/appsettings.json

```diff
@@ -2,6 +2,9 @@
   "ConnectionStrings": {
     "DefaultConnection": "Data source=tweetbook.db"
   },
+  "JwtSettings": {
+    "Secret": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
```

Comment:
Another way to do this such that no one would ever see your secrets, it if you enable user secrets in your project, move this line there, and then here you would only say instead of "aaa..." the name of the variable from the secret file.
Just a note to remember

Reply:
Good to know!